StudyBuddy/Controllers/ProfileController.cs

```diff
@@ -9,16 +9,20 @@ public class ProfileController : Controller
 {
     private readonly IUserManager _userManager; // Inject IUserManager
 
+
     public ProfileController(IUserManager userManager)
     {
         _userManager = userManager;
+        _userManager.LoadUsersFromCsv("test.csv");
```

Comment:
This will cause the file to be loaded everytime a request hits an endpoint on the ProfileController.